pkg/vm/engine/tae/tables/txnentries/mergeblocks.go

```diff
@@ -161,7 +163,19 @@ func (entry *mergeBlocksEntry) PrepareCommit() (err error) {
 		blks[i] = blk
 	}
 	var view *model.BlockView
+	skipped := 0
```

Comment:
This loop is VERY difficult to read.   You need to refactor it.

Reply:
Ok, I'll refactor mergeblocks lator.

Reply:
Now.   Or I won't approve.